.github/actions/run-tests/Dockerfile

```diff
@@ -0,0 +1,12 @@
+FROM golang:1.15-alpine
+
+# Add any build or testing essential system packages
+RUN apk add --no-cache build-base git
```

Comment:
@phillebaba once this gets merged, you'll be able to add the `libgit2-dev` dep here for https://github.com/fluxcd/source-controller/pull/213.

Reply:
Looks good!